apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -142,7 +142,7 @@ type ObjectStorageSpec struct {
 	// Name of a secret in the same namespace as the tempo Microservices custom resource.
 	//
 	// +optional
-	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Required
```

Comment:
To be honest I'm not sure if this marker changed anything at all (?)
Afaics it should have made the validation of an empty secret name obsolete (but it didn't).